resources/bank_accounts/models/bank_account.yml

```diff
@@ -17,16 +17,20 @@ allOf:
 
       signature_url:
         allOf:
-          - $ref: "../../../shared/attributes/signed_link.yml"
           - type: string
             nullable: true
+            description: A [signed link](#tag/Asset-URLs) to the signature image.
```

Comment:
For an explanation of why this works, see: https://github.com/OAI/OpenAPI-Specification/issues/2033